src/FakeItEasy/Configuration/FakeConfigurationManager.cs

```diff
@@ -100,12 +102,14 @@ private static Expression BuildArgumentThatMatchesAnything<TValue>()
 
         private static FakeManager GetFakeManagerCallIsMadeOn(ParsedCallExpression parsedCallExpression)
         {
-            if (parsedCallExpression.CallTarget == null)
-            {
-                throw new ArgumentException("The specified call is not made on a fake object.");
-            }
+            return parsedCallExpression.CallTarget == null
+                ? null

```

Comment:
Or rather, remove `GetFakeManagerCallIsMadeOn`, so we'd be left with:

```C#
public static void EnsureCallIsMadeOnAFake(ParsedCallExpression parsedCallExpression) =>
    Fake.GetFakeManager(parsedCallExpression.CallTarget);
```

and

```C#
public IReturnValueArgumentValidationConfiguration<T> CallTo<T>(Expression<Func<T>> callSpecification)
{
    Guard.AgainstNull(callSpecification, "callSpecification");

    var parsedCallExpression = this.callExpressionParser.Parse(callSpecification);
    var fake = Fake.GetFakeManager(parsedCallExpression.CallTarget);
    this.AssertThatMemberCanBeIntercepted(parsedCallExpression);

    var rule = this.ruleFactory.Invoke(parsedCallExpression);

    fake.AddRuleFirst(rule);

    return this.configurationFactory.CreateConfiguration<T>(fake, rule);
}
```

and

```C#
private IVoidArgumentValidationConfiguration CreateVoidArgumentValidationConfiguration(ParsedCallExpression parsedCallExpression)
{
    var fake = Fake.GetFakeManager(parsedCallExpression.CallTarget);
    var rule = this.ruleFactory.Invoke(parsedCallExpression);
    fake.AddRuleFirst(rule);

    return this.configurationFactory.CreateConfiguration(fake, rule);
}
```